.mypy.ini

```diff
@@ -0,0 +1,30 @@
+[mypy]
```

Comment:
Commented out things are to be enabled in future PRs. I'll fix up client.py in the next PR.